addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -215,7 +219,7 @@ export class LinkPlugin extends Plugin {
     }
 
     destroy() {
-        this.removeLinkShortcut();
+        this.detroyKeyboardCommand();
```

Comment:
- there's a typo in de**s**troy
- I understant the previous name was not the best, but this one is not that much more informative.
I would stick to the original name from the command service: `unregisterCommand` or some variation to make it clear that it's the "create link" command: `unregisterLinkCommand`.